src/DynamoPackagesUI/Utilities/PackageManagerCommands.cs

```diff
@@ -71,35 +71,51 @@ public dynamic PkgRequest
         /// <param name="downloadPath"></param>
         private void InstallPackage(string downloadPath)
         {
-            var firstOrDefault = Loader.LocalPackages.FirstOrDefault(pkg => pkg.ID == DownloadRequest.asset_id.ToString());
-            if (firstOrDefault != null)
+            if (!DynamoModel.IsTestMode)
```

Comment:
If you can pass a Mock PackageLoader, then you don't need to check if it's running in test mode. Ideally, this code should not be modified to handle test scenario.